src/Provider.tsx

```diff
@@ -439,24 +462,40 @@ export const RovingTabIndexContext = createContext<Context>({
  * If you do not pass an explicit value then the 'horizontal'
  * behaviour applies. You can change this direction value
  * at any time.
+ * @param {boolean} shouldLoopAround An optional boolean flag
+ * activating the tabindex navigation loop around. This will make
+ * the tabindex loop around to the other side of the list when
+ * trying to go past the first or last elements. You can
+ * change this direction value at any time.
```

Comment:
I would add a comment about this flag only applying if the roving tabindex is not being used on a grid.